brahma/data/payload/source/main.c

```diff
@@ -30,7 +30,7 @@ void main()
 	*((unsigned int*)0x080FFFC4) = 0x20046500;
 	*((unsigned int*)0x080FFFD8) = 0;
 
-	unsigned int* buf = 0x20400000;
+	unsigned int* buf = (void*)0x20400000;
```

Comment:
I suppose several lower level libraries will use void** anyway, so in case of (u8**) we can't avoid additional type casts. While void** itself is more flexible, though less safe